tfe/resource_tfe_organization_token_test.go

```diff
@@ -215,3 +261,36 @@ resource "tfe_organization_token" "regenerated" {
   force_regenerate = true
 }`, rInt)
 }
+
+func testAccTFEOrganizationToken_existsWithoutExpiry(rInt int) string {
```

Comment:
Not a big deal, but I believe we could just use `testAccTFEOrganizationToken_basic` in the place where we use this function.

Same for `testAccTFETeamToken_existsWithoutExpiry` and `testAccTFETeamToken_basic`

Reply:
I decided to repurpose these tests to check that a created token can have a `null` expiry, per your suggestion [here](https://github.com/hashicorp/terraform-provider-tfe/pull/844#discussion_r1202900712)!